scripts/main_app.py

```diff
@@ -0,0 +1,14 @@
+from pages import demo_playbooks, playbook_builder
+from streamlit import session_state as ss
+
+if "current_page" not in ss:
+    ss["current_page"] = "demo_playbooks"
```

Comment:
@pavangudiwada one more comment on consistency: either stick with `ss["foo"]` or `ss.foo` but don't mix and match unless there is a reason.